packages/medusa/src/loaders/plugins.ts

```diff
@@ -325,10 +325,12 @@ function registerApi(
   activityId: string
 ): Express {
   const logger = container.resolve<Logger>("logger")
-  logger.progress(
-    activityId,
-    `Registering custom endpoints for ${pluginDetails.name}`
-  )
+  const projectName =
+    pluginDetails.name === "project-plugin"
```

Comment:
todo: I was more talking about this one `"project-plugin"` I would suggest at the top of the file to have something like
```ts
const PROJECT_PLUGIN_NAME = "project-plugin"
```
and to use it in the different places instead